doc/source/guide/data_types/spectra.rst

```diff
@@ -97,7 +97,7 @@ To get rid of the noise, we could also clip low intensities by setting vmin.
 
 .. plot::
 
-    from matplotlib import pyplot as plt
+    import matplotlib.pyplot as plt
     import sunpy
     import sunpy.data.sample
     from sunpy.spectra.sources.callisto import CallistoSpectrogram
```

Comment:
If you ask for data, then yes... and if I remember right, in the documents it does a "expand" so it download the file before or after the main one...